### ISSUE TITLE
NTR: 2-ethylhydracrylic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8865]
New term request 2-ethylhydracrylic acid high in urine (elthat_131030185550):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000914
- label: Elevated urine 2-ethylhydracrylic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: 
- def: The amount of 2-ethylhydracrylic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8865
- synonyms:

**Comment by MickeySegal** [comment]
HP:6000914 was also assigned to "Single lesion" in https://github.com/obophenotype/human-phenotype-ontology/issues/10150#issuecomment-2210130511 and my comment there about the duplication was never answered.  The assignment of "Single lesion" to  HP:6000914 is in the public version of HPO.

In the public version of HPO I don't see a search result for "Elevated urine 2-ethylhydracrylic acid level".  Is there an HPO code assigned?
